util/docker/open3d-xvfb/tools/attach.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+. ./name.sh
```

Comment:
removing space? (applied to other scripts too.)

Reply:
DONE